packages/eth-edge/src/services/wallets.service.ts

```diff
@@ -25,6 +27,7 @@ export class WalletsService {
 
         const balance = await this.alchemyNodeService.getBalance(createWalletReq.address);
         const wallet = await this.findOne(createWalletReq.address, createWalletReq.userId);
+        this.transactionsService.backfil(wallet.address);
```

Comment:
Backfill is spelled with 2 l's, method needs to be awaited or else it is just scheduled in the event queue, and it should be moved up. You can consult the sequence diagram but it should be after gettng the balance but before getting the wallet.
```suggestion
        await this.transactionsService.backfill(wallet.address);
```